irr/include/SMesh.h

```diff
@@ -133,7 +133,7 @@ struct SMesh final : public IMesh
 	std::vector<u32> TextureSlots;
 
 	//! The bounding box of this mesh
-	core::aabbox3d<f32> BoundingBox{{0, 0, 0}};
+	core::aabbox3d<f32> BoundingBox{-1.0f, -1.0f, -1.0f, 1.0f, 1.0f, 1.0f};
```

Comment:
this is a fix but not the solution IMO.
we should be calculating the bounding box correctly